gen/builders/generator.go

```diff
@@ -42,9 +44,36 @@ type Generator struct {
 // genData is the generation data to stamp into vectors.
 // TODO in the future this should contain the commit of this tool and
 //  the builder api.
-var genData = GenerationData{
-	Source:  "script",
-	Version: "v0",
+var genData = []GenerationData{
+	{
+		Source:  "script",
+		Version: "v0",
```

Comment:
@raulk I imagine we will change this on build time.

Reply:
Yep, we need to figure that out.